test/txsim/run_test.go

```diff
@@ -23,6 +23,9 @@ import (
 )
 
 func TestTxSimulator(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping TestTxSimulator in short mode.")
+	}
```

Comment:
Inspired by the skip on `main`

Ref: https://github.com/celestiaorg/celestia-app/blob/44e8165cb37ccb4924cfd405493a67b6a2d7caa0/test/txsim/run_test.go#L28-L30